components/harvest/sources/new-timeentry-entry/new-timeentry-entry.mjs

```diff
@@ -0,0 +1,45 @@
+import harvest from "../../harvest.app.mjs";
+
+export default {
+  key: "harvest-new-timeentry-entry",
```

Comment:
The correct isn't `timeentry-entry`?

If yes, update the name of this file too!

```suggestion
  key: "harvest-new-time-entry",
```

Reply:
Yeah, it sounds funny, but it's a [Time Entry](https://help.getharvest.com/api-v2/timesheets-api/timesheets/time-entries/) object. So we are watching for new Time Entry entries :)